corehq/apps/sms/phonenumbers_helper.py

```diff
@@ -22,6 +22,11 @@ def parse_phone_number(number, region=None, failhard=True):
             return None
 
 
-def get_country_code(number, failhard=False):
+def get_country_code_and_national_number(number, failhard=False):
     parsed = parse_phone_number(number, failhard=failhard)
-    return parsed.country_code if parsed else None
+    if parsed:
+        country_code = parsed.country_code
+        before_national_number = "+%d" % parsed.country_code
```

Comment:
Are we sure that by the time 'number' gets here it still has the '+'? I'd be more comfortable if we didn't rely on the '+' being there, and if we instead set:

    national_number = strip_plus(number)[len(country_code):]

https://github.com/dimagi/commcare-hq/blob/sms-prefix-billing/corehq/apps/sms/util.py#L20